src/Momento.Sdk/Responses/CacheGetResponse.cs

```diff
@@ -1,11 +1,50 @@
 ﻿using Google.Protobuf;
 using Momento.Protos.CacheClient;
-
+using Momento.Sdk.Exceptions;
 namespace Momento.Sdk.Responses;
 
-public class CacheGetResponse : CacheGetResponseBase
+public class CacheGetResponse
 {
-    public CacheGetResponse(_GetResponse response) : base(response.Result, response.CacheBody)
+    public class Success : CacheGetResponse
+    {
+        public CacheGetStatus Status { get; }
+        protected readonly ByteString? value;
```

Comment:
I would have expected us to have sub classes for `Hit` and `Miss` here, and thus avoid the nullable stuff like we were doing elsewhere.

Reply:
You're totally right . . . I was just in a rhythm :-) Fixed in 0aa3060.

Reply:
That commit looks a great improvement but the field is still nullable here... it shouldn't need to be anymore, should it?